ts/utils/__tests__/dates.test.ts

```diff
@@ -78,3 +84,15 @@ describe("formatDateAsShortFormat", () => {
     expect(formatDateAsShortFormat(tt[0])).toEqual(tt[1]);
   });
 });
+
+describe("formatDateAsShortFormatUTC", () => {
+  const expected = "12/01/1983";
+  const toTest: ReadonlyArray<Date> = range(0, 23).map(
+    value =>
+      new Date(`1983-01-12T${value.toString().padStart(2, "0")}:00:00.000Z`)
+  );
+
+  toTest.forEach(tt => {
+    expect(formatDateAsShortFormatUTC(tt)).toEqual(expected);
```

Comment:
Since this behaviour is based on a localized string (see `"global.dateFormats.shortFormat"`), could it be useful to test even the case of locale changing? At the moment they have the same format, but if in future times we may change one of the localized formats this could break some behaviour

Reply:
I believe it's a good idea
Done https://github.com/pagopa/io-app/pull/3653/commits/70995a50e0f91c749a40b488f9a555cc6aac76ae